src/unix/stream.c

```diff
@@ -884,9 +884,16 @@ static void uv__write(uv_stream_t* stream) {
   QUEUE* q;
   uv_write_t* req;
   ssize_t n;
+  int cnt;
 
   assert(uv__stream_fd(stream) >= 0);
 
+  /* Prevent loop starvation when the consumer of this stream read as fast as (or faster than)
+   * we can write it. This `cnt` mechanism does not need to change even we switch to edge-triggered
+   * I/O.
+   */
+  cnt = 32;
```

Comment:
Can you name this `count` for consistency with `uv__read()` from this file?

Reply:
> BTW I think this should be created against the `v1.x` branch

The CONTRIBUTING.md says `As a rule of thumb, bug fixes go into the stable branch
while new features go into the master branch. The stable branch is effectively frozen; patches that change the libuv
API/ABI or affect the run-time behavior of applications get rejected`